java/semantickernel-core/src/main/java/com/microsoft/semantickernel/KernelDefault.java

```diff
@@ -31,19 +32,21 @@ public class KernelDefault implements Kernel {
     private final KernelConfig kernelConfig;
     private DefaultSkillCollection defaultSkillCollection;
     private final PromptTemplateEngine promptTemplateEngine;
-    @Nullable private SemanticTextMemory memory; // TODO: make this final
+    private final MemoryStore memoryStore;
```

Comment:
To mirror c# should be a SemanticTextMemory and default to NullMemory.getInstance()